app/services/exchange_rates/spot_exchange_rates_service.rb

```diff
@@ -0,0 +1,34 @@
+module ExchangeRates
+  class DataNotFoundError < StandardError; end
```

Comment:
Isn't this rewrite an existing error? I wonder if we could share this or just get rid of it :thinking: